src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -619,8 +619,7 @@ public SessionStateAliasEntry(string name, string definition, string description
         /// <param name="description">A description of the purpose of the alias.</param>
         /// <param name="options">Options defining the scope visibility, readonly and constant.</param>
         /// <param name="visibility"></param>
-        internal SessionStateAliasEntry(string name, string definition, string description,
-            ScopedItemOptions options, SessionStateEntryVisibility visibility)
+        internal SessionStateAliasEntry(string name, string definition, string description, ScopedItemOptions options, SessionStateEntryVisibility visibility)
```

Comment:
These look fairly long... we might be better off putting every parameter on its own line. 🤔

Reply:
This one seems to be just long enough to be on one line, but ok with either.